samples/zigbee/light_switch/src/main.c

```diff
@@ -472,10 +541,14 @@ void main(void)
 	/* Start Zigbee default thread */
 	zigbee_enable();
 
+#if CONFIG_NUS_CMD
+	/* Initalize NUS command service */
+	nus_cmd_init(on_nus_connect, on_nus_disconnect, commands);
+#endif
+
 	LOG_INF("ZBOSS Light Switch example started");
 
 	while (1) {
-		dk_set_led(RUN_STATUS_LED, (++blink_status) % 2);
-		k_sleep(RUN_LED_BLINK_INTERVAL);
+		k_sleep(K_FOREVER);
```

Comment:
We may consider aligning this change other examples, so in case of a Zigbee-only application, a user will not wait for the LED1 to start blinking.